packages/react-router/lib/server-runtime/server.ts

```diff
@@ -169,14 +169,30 @@ export const createRequestHandler: CreateRequestHandlerFunction = (
     if (!_build.ssr) {
       // Decode the URL path before checking against the prerender config
       let decodedPath = decodeURI(normalizedPath);
-      let decodedPathWithoutBasename = stripBasename(
-        normalizedPath,
-        normalizedBasename,
-      );
 
-      // handle root path
-      if (decodedPathWithoutBasename === "/") {
-        decodedPathWithoutBasename = "";
```

Comment:
I don't think we needed this check because we just add the trailing slash back on below in `!_build.prerender.includes(decodedPath + "/")`